diff --git a/node_modules/@pauliescanlon/gatsby-mdx-routes/src/components/MdxRoutes/MdxRoutes.js b/node_modules/@pauliescanlon/gatsby-mdx-routes/src/components/MdxRoutes/MdxRoutes.js
index 9b3ca23..a00ebfa 100644
--- a/node_modules/@pauliescanlon/gatsby-mdx-routes/src/components/MdxRoutes/MdxRoutes.js
+++ b/node_modules/@pauliescanlon/gatsby-mdx-routes/src/components/MdxRoutes/MdxRoutes.js
@@ -10,11 +10,13 @@ export const MdxRoutes = ({ children }) => {
           allMdx {
             edges {
               node {
+                fileAbsolutePath
                 fields {
                   slug
                 }
                 frontmatter {
                   navigationLabel
+                  order
                 }
               }
             }
@@ -26,6 +28,8 @@ export const MdxRoutes = ({ children }) => {
           return {
             navigationLabel: data.node.frontmatter.navigationLabel,
             slug: data.node.fields.slug,
+            order: data.node.frontmatter.order,
+            fileAbsolutePath: data.node.fileAbsolutePath,
           }
         })
         return children(mdxData)
