diff --git a/node_modules/jest-environment-puppeteer/lib/global.js b/node_modules/jest-environment-puppeteer/lib/global.js
index 81ba353..a1f029a 100644
--- a/node_modules/jest-environment-puppeteer/lib/global.js
+++ b/node_modules/jest-environment-puppeteer/lib/global.js
@@ -26,7 +26,7 @@ async function setup(jestConfig = {}) {
   }
 
   process.env.PUPPETEER_WS_ENDPOINT = browser.wsEndpoint();
-  if (jestConfig.watch || jestConfig.watchAll) return;
+  //if (jestConfig.watch || jestConfig.watchAll) return;
 
   if (config.server) {
     try {
@@ -52,7 +52,7 @@ async function setup(jestConfig = {}) {
 }
 
 async function teardown(jestConfig = {}) {
-  if (!jestConfig.watch && !jestConfig.watchAll) {
+  //if (!jestConfig.watch && !jestConfig.watchAll) {
     await (0, _jestDevServer.teardown)();
     const config = await (0, _readConfig.readConfig)();
 
@@ -61,5 +61,5 @@ async function teardown(jestConfig = {}) {
     } else {
       await browser.close();
     }
-  }
+  //}
 }
